ee/ucp/kubernetes/install-cni-plugin.md

```diff
@@ -1,98 +1,117 @@
 ---
-title: Install a CNI plugin
+title: Install an Unmanaged CNI plugin
 description: Learn how to install a Container Networking Interface plugin on Docker Universal Control Plane.
-keywords: ucp, cli, administration, kubectl, Kubernetes, cni, Container Networking Interface, flannel, weave, ipip, calico
+keywords: ucp, Kubernetes, cni, Container Networking Interface, flannel, weave, calico
 ---
 
-For Docker Universal Control Plane, [Project Calico](https://docs.projectcalico.org/v3.0/introduction/) 
-provides the secure networking functionality for the container communication with Kubernetes.
+For Docker Universal Control Plane, [Project
+Calico](https://docs.projectcalico.org/v3.7/introduction/) provides the secure
+networking functionality for the container to container communication within
+Kubernetes. The Universal Control Plane handles the lifecycle of Project Calico,
+installing and upgrading the project as part of UCP. Additionally, the Project
+Calico deployment installed is fully supported, with Docker Support providing
+guidance on the [CNI components](https://github.com/projectcalico/cni-plugin).
 
-Docker EE supports Calico and installs the 
-built-in [Calico](https://github.com/projectcalico/cni-plugin) plugin, but you can override that and 
-install a Docker certified plugin.
+At install time, the Universal Control Plane can be configured to not install
```

Comment:
@uday-shetty  I believe this statement to be spot on. Think would be good to have some PM or even CS eyes on it though (StratDev doesn't really own our support policy / statements). 
Otherwise, agree with the gist and positioning, LGTM.

cc @david-yu / @mklamb for visibility / comment.

Reply:
@nerdalert @gnomestar LGTM as well. Thanks!

Reply:
I've updated the statement it my latest commit. Do you direct link to where the certified CNIs live on Docker Hub?

Reply:
@david-yu or @mklamb - circling back to this PR and have started making my edits. Can we please get your input on the statements of support included in this file?

Reply:
LGTM

Reply:
LGTM

Reply:
> I've updated the statement it my latest commit. Do you direct link to where the certified CNIs live on Docker Hub?

https://hub.docker.com/search?q=&type=plugin&category=network